registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/spi/packetmanager/PacketInfoManager.java

```diff
@@ -199,6 +199,15 @@ public void saveManualAdjudicationData(List<String> uniqueMatchedRefIds, String
 	 */
 	public List<RegBioRefDto> getBioRefIdByRegId(String regId);
 
+	/**
+	 * Gets the bio ref ids list by bioRefId.
+	 *
+	 * @param bioRefId
+	 *            the bio ref id
+	 * @return all the bioRefIds dto
+	 */
+	public List<RegBioRefDto> getRegBioRefDataByBioRefId(List<String> bioRefId);
```

Comment:
Since it takes in a list, I think this method should be renamed to getRegBioRefDataByBioRefIds

Reply:
ok